src/lib/config.ts

```diff
@@ -1670,68 +1699,13 @@ const parseConfigHelper = (pconf, parseobj) => {
     return parseobj
 }
 
-// Listen for changes to the storage and update the USERCONFIG if appropriate.
-// TODO: BUG! Sync and local storage are merged at startup, but not by this thing.
-browser.storage.onChanged.addListener((changes, areaname) => {
-    if (CONFIGNAME in changes) {
-        const { newValue, oldValue } = changes[CONFIGNAME]
-        const old = oldValue || {}
-
-        function triggerChangeListeners(key, value = newValue[key]) {
```

Comment:
I think this function should probably stay, otherwise `addChangeListener` changeListeners won't be called.